doc/README.md

```diff
@@ -434,7 +434,7 @@ Graph
 Flame Graph
```

Comment:
Indentation is different than for "Graph" and other items above. Is this intentional?

Reply:
Made all suggested changes.